src/components/CardInfo.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import '../App.css';
+import PropTypes from 'prop-types';
+import { Container, Row, Col } from 'react-bootstrap';
+
+const CardInfo = ({ geolocation }) => {
+  const { ip, location, isp } = geolocation;
+  const { region, country, postalCode, timezone } = location;
+  return (
+    <Container className="card-container" style={{ zIndex: '999' }}>
```

Comment:
> Would it make sense to add `zIndex: 999` to the `card-container` class instead of adding it directly to the component styles?

I used `z-index-3` now, but now sure whether it would work as I haven't done anything relating to style at the moment for the cardInfo on the browser